README.md

```diff
@@ -24,9 +24,9 @@ App| Description
 ---|----------------------------------------------------------------------------|---
 [hello_serial](hello_world/serial) | The obligatory Hello World program for Pico (Output over serial version)   |
 [hello_usb](hello_world/usb) | The obligatory Hello World program for Pico (Output over USB version)      | https://rptl.io/pico-hello-usb
-[blink](blink) | Blink an LED on and off. Works on both boards with regular LEDs and Pico W | https://rptl.io/pico-blink
-[blink_simple](blink_simple) | Blink an LED on and off. Does not work on Pico W.                          | https://rptl.io/pico-blink
-[picow_blink](pico_w/wifi/blink) | Blinks the Pico W on-board LED (which is connected via the WiFi chip).     | http://rptl.io/pico-w-blink
+[blink](blink) | Blink a LED on and off. Works on both boards with regular LEDs and boards like Pico W where the led is connected via the Wi-Fi chip | https://rptl.io/pico-blink
+[blink_simple](blink_simple) | Blink a LED on and off. Does not work on boards like Pico W where the led is connected via the Wi-Fi chip. | https://rptl.io/pico-blink
```

Comment:
Presumably `https://rptl.io/pico-blink` can't be pointing at **both** `blink` and `blink_simple` - do we know which it is?